clar2wasm/src/words/functions.rs

```diff
@@ -300,11 +299,9 @@ mod tests {
             StacksEpochId::Epoch20,
         );
 
-        // Latest Epoch and clarity version
-        crosscheck("(define-private (index-of?) (ok u0))", Err(()));
-        crosscheck("(define-private (index-of) (ok u0))", Err(()));
-        crosscheck("(define-private (element-at?) (ok u0))", Err(()));
-        crosscheck("(define-private (element-at) (ok u0))", Err(()));
+        crosscheck_expect_failure("(define-private (index-of?) (ok u0))");
+
+        crosscheck_expect_failure("(define-private (element-at?) (ok u0))");
```

Comment:
Why would you remove those tests???

Reply:
re-added the test variant for the function alias.